OpenGL/Constructs/VAO.cs

```diff
@@ -36,6 +38,29 @@ public class VAO<T1, T2> : GenericVAO
         where T1 : struct
         where T2 : struct
     {
+        public VAO(ShaderProgram program, VBO<T1> vbo1, string attribName, VBO<T2> elementArray)
+    : base(program, false)
+        {
+            GenericVAO.IGenericVBO[] vbos = new IGenericVBO[2];
+            vbos[0] = new GenericVBO<T1>(vbo1, attribName);
+            vbos[1] = new GenericVBO<T2>(elementArray, "");
+
+            Init(vbos);
+        }
+
+        public VAO(ShaderProgram program, VBO<T1> vbo1, string[] attribNames, VBO<T2> elementArray)
+    : base(program, false)
+        {
+            if (attribNames.Length != 1) throw new Exception(string.Format("Expected an array of 1 names, but instead got {0}.", attribNames.Length));
+
+            GenericVAO.IGenericVBO[] vbos = new IGenericVBO[2];
+            vbos[0] = new GenericVBO<T1>(vbo1, attribNames[0]);
+            vbos[1] = new GenericVBO<T2>(elementArray, "");
+
+            Init(vbos);
+        }
+
+        [Obsolete("Constructors that doesn't specify the element array type are deprecated. Please use a constructor that specifies the element arrays type.")]
```

Comment:
I think this should be: "Constructors that don't specify the element array type are deprecated. Please use a constructor that specifies the element array type."